markdown/apigee-apim-integration/apigee-apim-integration.md

```diff
@@ -0,0 +1,558 @@
+author: Swen-Helge Huber
+summary:
+id: apigee-apim-integration
+tags: iguide
+categories: solace
+environments: Web
+status: Published
+feedback link: https://github.com/SolaceDev/solace-dev-codelabs/blob/master/markdown/apigee-apim-integration
+
+# Unified APIM: Integrating Solace Event Portal with apigee Platform and Developer Portal 
+
+## Overview
+
+Duration: 0:05:00
+
+The blog ["Take Event APIs to where your Apigee APIs are …"](http:todo.com/) introduced an integration between the Google Apigee and Solace Platforms.
+
+In this codelab we walk you through the steps that enable the developer experience for event APIs demonstrated in the blog.
+
+The overall solution looks like this:
+![Solution Overview](img/overview.png)
+
+Let’s look at all the components involved:
+* apigee Dev Portal: the developer portal that developer interact with to find APIs and register their apps
+* App: the application that the developer creates. We simulated this app with postman in the sections above.
+* PubSub+ Broker: the event broker that the event API is accessible on. As it acts similarly to an API Gateway, we could also call it an “Event Gateway”. 
+* Google apigee API Gateway: the API gateway hosting all the required API Proxies for the integration (more on these later)
+* Solace Event Portal: the event governance platform that allows to manage Events, Event APIs an Event API Products.
+* apigee Platform: the API management back office
+
+As you can see the interactions flow through the apigee gateway as we chose to implement the integration completely in apigee:
+* The integration provides a REST API for Discovery of Event API Products and to subsequently import these to the Dev Portal and apigee platform. This includes uploading AsyncAPIs to the developer portal as well as creating apigee API Products that represent Event API Products
```

Comment:
A bit nit picking - but I would suggest to change this to AsyncAPI specs instead of `AsyncAPIs`. Multiple occurrences in the markdown

Reply:
Done. Only kept in places that refer to items labelled "AysncAPI" in the Apigee Dev Portal